shared/cfg/guest-os/Linux/Fedora/31.cfg

```diff
@@ -0,0 +1,107 @@
+- 31:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
```

Comment:
ppc64 is not available for F31...

Reply:
yeah,since fedora29,sorry for having not cleaned up this file.